src/App.jsx

```diff
@@ -14,70 +15,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const isGoodSelected = good => good === selectedGood;
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  const deletGood = (currGood, goodOnClick) => {
+    if (currGood === goodOnClick) {
+      return '';
+    }
```

Comment:
```suggestion
  const handleGoodClick = (good) => {
   setSelectedGood(isGoodSelected(good) ? good : '');
          }
```

Looks more clear, doesn't it?)